.github/workflows/build.yml

```diff
@@ -21,11 +21,16 @@ jobs:
         - '7.0.2.3'
         - '7.1.0'
         - '7.2.0'
+        - '8.0.0'
         exclude:
         - ruby-version: 2.7
-          rails: '7.2.0'
+          rails:
+            - '7.2.0'
+            - '8.0.0'
         - ruby-version: '3.0'
-          rails: '7.2.0'
+          rails:
+            - '7.2.0'
+            - '8.0.0'
```

Comment:
Rails 8.0.0 only supports Ruby >= 3.2.0.
https://rubygems.org/gems/rails/versions/8.0.0

So we need to add the Rails 8.0 and Ruby 3.1 matrix to the `exclude` list.